frontend/src/components/object/DeleteObjectButton.vue

```diff
@@ -42,8 +42,15 @@ const confirmDelete = () => {
       rejectLabel: 'Cancel',
       accept: async () => {
         try {
-          await objectStore.deleteObjects(props.ids, props.versionId);
-          emit('on-deleted-success', props.versionId);
+          const res = await objectStore.deleteObjects(props.ids, props.versionId);
+          res?.forEach((ele, index)=>{
```

Comment:
Formatting/spacing with the arrow.

While this does technically work, this does have the issue of blocking immediacy of responses. While this is a delete so shouldn't be affected, consider if you were instead uploading one small and one very large file. Promise.allSettled will force both alerts. big and small, to stall out and wait for both to be complete before you ever get the alert, even though the small file may have finished a long time ago. It would be more ideal to consider instead, creating an array of singular objectStore.deleteObject calls instead; then the component here would have agency over the entire asynchronous chain of actions on each individual file (and dispatching appropriately) instead of shunting everything into a promiseSettled and making everything have to wait until hard completion.